src/components/MenuButton/MenuButton.tsx

```diff
@@ -179,8 +179,8 @@ const MenuButton: VibeComponent<MenuButtonProps> & {
 
     const onMenuDidClose = useCallback(
```

Comment:
This event is called when you click on a MenuItem, but not on a SubMenu or SplitMenu Item, which is exactly what we want. Not sure why it reacted only to keyboard event though